packages/suite/src/components/suite/DeviceDisplay.tsx

```diff
@@ -105,7 +109,8 @@ export const DeviceDisplay = ({ address, network, valueDataTest }: DeviceDisplay
         addressDisplayType === AddressDisplayOptions.CHUNKED &&
         !unavailableCapabilities?.chunkify &&
         valueDataTest !== '@xpub-modal/xpub-field' &&
-        network !== 'cardano';
+        ((network !== 'cardano' && network !== 'solana') ||
+            (network !== 'cardano' && valueDataTest === '@modal/confirm-address/address-field'));
```

Comment:
Haven't tried but couldn't it be simplified like this?

```suggestion
        network !== 'cardano' && (network !== 'solana' || valueDataTest === '@modal/confirm-address/address-field';
```

Reply:
Yes it's working, thanks